BDArmory/Ammo/PooledBullet.cs

```diff
@@ -1275,8 +1276,7 @@ bool BulletHitAnalysis(RaycastHit hit, float period)
                             // If we don't, I.E. the round isn't completely eroded, we decrease
                             // the velocity by a max of 5%, proportional to the adjustedPenRatio
                             //impactVelocity = impactVelocity * (0.95f + 0.05f * adjustedPenRatio);
-                            adjustedPenRatio *= 0.05f;
-                            adjustedPenRatio += 0.95f;
+                            adjustedPenRatio = 0.95f + 0.05f * adjustedPenRatio;
```

Comment:
This should be `*=`. It should be scaling `adjustedPenRatio` by a factor just under 1, not resetting it to just under 1.